modules/ROOT/pages/questions/question-17.adoc

```diff
@@ -0,0 +1,13 @@
+17. https://ephocks.atlassian.net/browse/TINY-4603[TINY-4603] notes
```

Comment:
@HAFRMO @ltrouton TINY-4603

Reply:
These option names were changed to make them more consistent with the other `link` and `autolink` options available. They are purely name changes. The functionality of these options and the possible values they can be given has not changed